test/bats/test.bats

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bats
+load helpers
+
+BATS_TESTS_DIR=test/bats/tests
+WAIT_TIME=120
+SLEEP_TIME=1
+
+@test "osm pod has been deployed and is running." {
+    run wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl wait --for=condition=Ready --timeout=60s pod -l app=osm-controller"
```

Comment:
@SanyaKochhar This should be failing right now as the latest chart has a known bug: https://github.com/openservicemesh/osm/issues/1829

Reply:
doesn't seem to be failing for me, could it be that whatever is causing this in osm hasn't propagated to osm-azure charts yet?

Reply:
We can ignore this bug for now. 
1. The chart in main is using `0.4.0` image tag so this bug doesnt happen on a fresh cluster. only happens on `0.4.1`.
2. with `0.4.0` image tag, this bug only happens on a reinstall of the chart.